experimental/entrypoint.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+echo "Running pycodestyle."
+pycodestyle --exclude=vendor --ignore=E402,W503,E501 target_finder_model generate scripts_tf test
+
+echo "Building data."
+python scripts_generate/build.py
+
+echo "Creating TF-Records."
+python scripts_tf/create_tf_records.py
+
+cp -r model_data/records /host/model_data
+
+echo "Creating label map."
+python scripts_tf/create_label_map.py
+
+echo "Training classification model."
+#python scripts_tf/train_clf.py
```

Comment:
Should this be commented out?

Reply:
Not exactly, I'll add a blurb on the readme about experimental code